scripts/geth.sh

```diff
@@ -1,35 +1,88 @@
 #!/usr/bin/env bash
 
 ORIGARGS=("$@")
+TMP_FOLDER=$(pwd)/tmp
+IPC_PATH="$TMP_FOLDER/ipc"
 
 helpFunction() {
 	echo "Usage: $0 [start|stop] [background]"
 	exit 1 # Exit script after printing help
 }
 
+getOS(){
+    case "$OSTYPE" in
+      solaris*) OS="SOLARIS" ;;
+      darwin*)  OS="OSX" ;;
+      linux*)   OS="LINUX" ;;
+      bsd*)     OS="BSD" ;;
+      msys*)    OS="WINDOWS" ;;
+      cygwin*)  OS="ALSO WINDOWS" ;;
+      *)        OS="unknown: $OSTYPE" ;;
+    esac
+}
+getDownloadLink(){
+    case "$OS" in
+      SOLARIS*) LINK="-" ;;
+      OSX*)  LINK="https://gethstore.blob.core.windows.net/builds/geth-darwin-amd64-1.10.19-23bee162.tar.gz" ;;
+      LINUX*)   LINK="https://gethstore.blob.core.windows.net/builds/geth-linux-386-1.10.19-23bee162.tar.gz" ;;
```

Comment:
This geth bin was not working for me and I used https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.10.19-23bee162.tar.gz for local testing.